src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQSet.cpp

```diff
@@ -49,55 +50,55 @@ const std::string& UDQScalar::wgname() const {
 
 void UDQScalar::assign(double value) {
     this->m_value = value;
-    this->m_defined = true;
+    this->m_defined = std::isfinite(value);
```

Comment:
Is `value = 0.0` a well-defined value here?  If not, you can actually use
```C++
this->m_defined = std::isnormal(value);
```
instead.

Reply:
0.0 is a perfectly legitimate value in this case; so `std::isnormal( )` is not an option.